app/src/main/res/values/colors.xml

```diff
@@ -218,6 +218,7 @@
     <color name="contrast_text_normal_theme">@color/contrast_text_light_theme</color>
     <color name="caption_text_normal_theme">@color/caption_text_light_theme</color>
     <color name="foundation_normal_theme">@color/foundation_light_theme</color>
+    <color name="foundation_contrast_normal_theme">@color/foundation_dark_theme</color>
```

Comment:
Based on some of the points made above, I am wondering if we want to add this variable over just using `top_site_pin_icon_background_tint`. Technically the colours are inverted instead of contrast. I would perhaps reserve adding more of these foundation variables until we figure out a component system where we do map out components colors to appropriate variables.